usr/sbin/registercloudguest

```diff
@@ -144,7 +144,43 @@ def setup_registry(registration_target, clean='registry'):
         utils.get_credentials_file(registration_target)
     )
     registry_fqdn = registration_target.get_registry_FQDN()
+
+    with_docker = True
+    if not utils.is_docker_present():
+        with_docker = False
+        utils.clean_registries_conf_docker(registry_fqdn)
```

Comment:
We determined that the config file is not present, why call clean? That only triggers an additional message to the logfile """'Unable to read "{0}", cleanup not possible'.format(DOCKER_CONFIG_PATH)""".
If there's no docker then we only write the log message. Or am I missing something?

Reply:
yes this doesn't make any sense, thanks for spotting it. I tricked myself here. The idea was to cleanup a left over config setup when docker was uninstalled. But we detect that state by checking for the config file. I also realized when you uninstall docker the config gets moved as .rpmsave. So yes the cleanup here is stupid. Sorry I'm going to drop this

Reply:
done